src/test/java/api/requests/ItemsInTransitReportTests.java

```diff
@@ -582,6 +604,15 @@ private InventoryItemResource createSmallAngryPlanet() {
       itemsFixture.thirdFloorHoldings());
   }
 
+  private InventoryItemResource createSmallAngryPlanet(UUID locationId, int barcode) {
```

Comment:
I think `createSmallAngryPlanetCopy` would probably describe what this method does better.

Reply:
renamed